pkg/envtest/crd.go

```diff
@@ -340,6 +348,33 @@ func renderCRDs(options *CRDInstallOptions) ([]client.Object, error) {
 	return res, nil
 }
 
+func modifyConversionWebhooks(crds []client.Object, webhookOptions WebhookInstallOptions) error {
+	// generate host port.
+	hostPort, err := webhookOptions.generateHostPort()
+	if err != nil {
+		return err
+	}
+	url := pointer.StringPtr(fmt.Sprintf("https://%s/convert", hostPort))
```

Comment:
is conversion always on `/convert`?  Should we be keeping the user's configured path, and just messing with the host-port part?

Reply:
Yeah by convention it's always `/convert`, see  https://github.com/kubernetes-sigs/controller-runtime/blob/b704f447ea7c8f7059c6665143a4aa1f6da28328/pkg/builder/webhook.go#L139-L141

Reply:
theoretically it could be not convert if you don't go through the builder, but I'm not sure how much we need to worry about that ATM.